service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocmosisDocumentGeneratorServiceTest.java

```diff
@@ -27,35 +30,25 @@
 
     private String tornadoUrl = "http://tornado:5433";
 
+    @Captor
+    ArgumentCaptor<HttpEntity<DocmosisRequest>> argumentCaptor;
+
     @Test
+    @SuppressWarnings("unchecked")
     void shouldInvokesTornado() {
         Map<String, String> placeholders = getTemplatePlaceholders();
 
-        when(restTemplate.exchange(eq(tornadoUrl + "/rs/render"), eq(HttpMethod.POST), any(), eq(byte[].class)))
-            .thenReturn(tornadoResponse);
+        when(restTemplate.exchange(eq(tornadoUrl + "/rs/render"), eq(HttpMethod.POST), argumentCaptor.capture(),
+            eq(byte[].class))).thenReturn(tornadoResponse);
 
         byte[] expectedResponse = {1, 2, 3};
         when(tornadoResponse.getBody()).thenReturn(expectedResponse);
 
         DocmosisDocument docmosisDocument = createServiceInstance().generateDocmosisDocument(placeholders, C6);
         assertThat(docmosisDocument.getBytes()).isEqualTo(expectedResponse);
-    }
-
-    @Test
-    void shouldThrowNullPointerIfAnInValidDocmosisTemplateEnumIsNotProvided() {
```

Comment:
Do you think there was no value in the test?

Reply:
Yeah, it's just the behaviour of Java to throw a null pointer if an expected variable is null. Did not think it was required.

Reply:
Yeah, no point of testing default behaviour.